gems/aws-sdk-core/lib/aws-sdk-core/log/param_filter.rb

```diff
@@ -11,33 +11,39 @@ class ParamFilter
       # end
 
       def initialize(options = {})
-        @filters = Set.new(SENSITIVE + Array(options[:filter]))
+        filters = options[:filter] || {}
```

Comment:
To not break current customers, we should "splat" these to all services in the hash if it's an array.